CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.9.2 (2020-02-07)
+
+- Fixed the primary entrypoint to export `SkyIntlNumberFormatStyle`. [#54](https://github.com/blackbaud/skyux-i18n/pull/54)
```

Comment:
To John's point on the builder PR, let's split up entry point to be two words.

Reply:
Done!